### ISSUE TITLE
Remove the JSON key import option from the setup wizard

### ISSUE THREAD

**Issue by pasqu4le** [serokell/tezos-packaging#583]
# Clarification and motivation

The `tezos-setup` wizard has several key import options, including a JSON one for test networks.

This was needed as it was the way to use an account with some funds from a faucet, but that's no longer the case, see #525 

This switch happened quite a while ago, however importing a key from a JSON file is still supported by Octez and so it's worth keeping. However, it shouldn't be given as only useful for faucets and be allowed on `mainnet` as well. 

# Acceptance criteria

~~The `tezos-setup` wizard no longer has a JSON key import option~~

In the setup wizard, the option to import a key from a JSON source is:
1. available for `mainnet` and not only testnets
2. given as yet another option, without mentioning the faucet

**Comment by pasqu4le** [comment]
On second thought, while this option was mainly used for the faucet, it's still possible to produce these JSON files and Octez is still able to use them.

With this in mind it's worth keeping this as a choice, however documentation and messages in the setup wizard need to updated. So, I'm changing this issue's acceptance criteria now.

**Comment by pasqu4le** [comment]
Looking further into it today, my current acceptance criteria are misguided, I think.

On a testnet, the "activate account" command help text is:
```
  activate account <new> with <activation_key> [-f --force] [--encrypted]
    Register and activate an Alphanet/Zeronet faucet account.
    <new>: new secret_key alias
    <activation_key>: Activate an Alphanet/Zeronet faucet account from the JSON (file or directly inlined).
    -f --force: overwrite existing secret_key
    --encrypted: encrypt the key on-disk
```
Note: this was taken from `limanet`, but I doubt that changed recently because only Alphanet/Zeronet are mentioned even tho JSON files were used in much more recent test networks too.

So, I don't think that we should stop mentioning the faucets, because:
1. that seems to have been the main purpose of keys imported from JSON files from the very beginning
2. they are mentioned in the texts of `octez-client` too

Moreover, it doesn't look like the command above is available on `mainnet`.
There the help text points at a different command altogether: 
```
  activate fundraiser account <name> with <code> [-D --dry-run]
    Activate a fundraiser account.
    <name>: existing public key hash alias
    <code>: Activation code obtained from the Tezos foundation.
    -D --dry-run: don't inject the operation, just display it
```
which doesn't even use JSON or JSON files AFAICT.

So, we cannot extend the ability to import a key from JSON to mainnet, as that appears to be forbidden there.

With that, both of the current acceptance criteria are null and void.
Lastly, it's still true that there might be some use for this import method for faucets of testnets in the future, so it doesn't seem worth removing this option from the `tezos-setup` wizard.

In conclusion, I'm closing this issue without making any changes.
